src/validation.cpp

```diff
@@ -5177,6 +5178,18 @@ CChainState& ChainstateManager::InitializeChainstate(CTxMemPool& mempool, const
     return *to_modify;
 }
 
+std::optional<std::reference_wrapper<const AssumeutxoData>> ExpectedAssumeutxo(
```

Comment:
In commit "chainparams: add allowed assumeutxo values" (647c91957f99f5b9aed790e49941e3370fc3c0b2)

Should replace `optional<reference_wrapper<T>>` with `T*`. The advantage of using references instead of pointers is that references are not optional, so you don't have to check for null. If you take reference and make it optional, you are just verbosely emulating a pointer.